### PR TITLE
Added jQuery sortable to let user rearange fields with drag & drop

### PR DESCRIPTION
**Pull request by DebVortex**
Currently the test coverage of dragable-elements.js is pretty bad but there is a problem with testing promises I was not able to solve, yet. Therefore I would propose to go on for now and get back later on testing.

### CODE REVIEW

File: app/components/dragable-elements.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            toleranceType: "pointer",
+            update: function(event, ui) {
+                if (ui.item.find('.field').hasClass('fieldset') &&
+                        ui.item.parents().eq(2).hasClass('fieldset')) {
+                    hopperOutlet.find('.sub-elements').sortable('cancel');
+                }
+                var order = {};
+                Ember.$('.field').each(function(index) {
+                    var element = Ember.$(this);
+                    var parentNode = element.parents().eq(3);
```
Comment (rm--) [inline app/components/dragable-elements.js:23]: Why 3?

Comment (DebVortex) [inline app/components/dragable-elements.js:23]: $().parents() returns all elements upwards till the root node, starting with the current one. The element 0 is the current one. 1 is the div.sub-element we are using to connect with. 2 is a div, added by ember. Therefore, getting the hopper-outlet or the parent element we need to use the 3.

Comment (rm--) [inline app/components/dragable-elements.js:23]: Please add this explanation as comment. Perhabs it's better to define a constant and add the comment there.

File: tests/unit/components/dragable-elements-test.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  // needs: ['component:foo', 'helper:bar']
+});
+
+test('removeFormElements', function(assert) {
+  assert.expect(2);
+  var component = this.subject();
+  var obj = fakeObject.create();
+  var formElements = new Array('Foo', 'Bar');
+  obj.set('formElements', formElements);
+  assert.equal(obj.get('formElements').length, 2);
```
Comment (rm--) [inline tests/unit/components/dragable-elements-test.js]: Please use a constant for L45 and L50.

File: tests/unit/components/dragable-elements-test.js [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+});
+
+test('addToFormElements', function(assert) {
+  assert.expect(2);
+  var component = this.subject();
+  component.store = fakeStore();
+  var obj = fakeObject.create();
+  var elements = new Array('Foo', 'Bar');
+  obj.set('formElements', elements);
+  assert.equal(obj.get('formElements').length, 2);
```
Comment (rm--) [inline tests/unit/components/dragable-elements-test.js]: Please use a constant for L56 and L62.

File: app/components/dragable-elements.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+        });
+    },
+
+    removeFormElements: function(obj) {
+        // remove all formElements from the given object
+        var formElements = obj.get('formElements').toArray();
+        obj.get('formElements').removeObjects(formElements);
+    },
+
+    addToFormElements: function(elementId, addTo) {
```
Comment (rm--) [inline app/components/dragable-elements.js]: Rename `addTo` to `parent`.

**Pull request merged**